bot/cogs/bigbrother.py

```diff
@@ -391,8 +425,29 @@ def _parse_infraction_time(infraction: dict) -> struct_time:
         # !nominate command does not show up under "BigBrother" in the help embed, but under
         # the header HelperNomination for users with the helper role.
 
+        member = ctx.guild.get_member(user.id)
+
+        if member and any(role.id in STAFF_ROLES for role in member.roles):
+            await ctx.send(f":x: {user.mention} is already a staff member!")
+            return
+
         channel_id = Channels.talent_pool
 
+        # Update watch cache to avoid overwriting active nomination reason
+        await self.update_watched_users()
+
+        if user.id in self.watched_users:
+            if self.watched_users[user.id].id == Channels.talent_pool:
+                prefix = "Additional nomination: "
+            else:
+                # If the user is being watched in big-brother, don't add them to talent-pool
+                await ctx.send(f":x: Can't add {user.mention} to the talent-pool at this moment")
```

Comment:
This could be more specific

Reply:
Yes, I was debating with myself what kind of message to put there. Since the command is available from helper and up, this is currently the only command that reveals information about a user's big brother watch status that's available to non-mod+ roles. 

However, since this is the only way this message can appear anyway, it's not hard to deduce the implication of it anyway, so stating it explicitly probably won't hurt.

Reply:
I've changed the message.